packages/components/nodes/memory/MotorheadMemory/MotorheadMemory.ts

```diff
@@ -0,0 +1,96 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { ICommonObject } from '../../../src'
+import { MotorheadMemory, MotorheadMemoryInput } from 'langchain/memory'
+
+class MotorMemory_Memory implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Motorhead Memory'
+        this.name = 'motorheadMemory'
+        this.type = 'MotorheadMemory'
+        this.icon = 'memory.svg'
+        this.category = 'Memory'
+        this.description = 'Remembers previous conversational back and forths directly'
+        this.baseClasses = [this.type, ...getBaseClasses(MotorheadMemory)]
+        this.inputs = [
+            {
+                label: 'Base URL',
+                name: 'baseURL',
+                type: 'string',
+                optional: true,
+                description: 'To use the online version, leave the URL blank. More details at https://getmetal.io.'
+            },
+            {
+                label: 'Memory Key',
+                name: 'memoryKey',
+                type: 'string',
+                default: 'chat_history'
+            },
+            {
+                label: 'Session Id',
+                name: 'sessionId',
+                type: 'string',
+                description: 'if empty, chatId will be used automatically',
+                default: '',
+                additionalParams: true,
+                optional: true
+            },
+            {
+                label: 'API Key',
+                name: 'apiKey',
+                type: 'string',
+                optional: true
```

Comment:
can we add `additionalParams: true` and `description: 'Only needed when using hosted solution https://getmetal.io'`